### PR TITLE
Add eyra-test-scan Claude Code skill

### PR DESCRIPTION
**Pull request by irisvanderveen**
## Summary

- Voegt de `eyra-test-scan` skill toe voor Claude Code
- Helpt bij het beoordelen van testdekking op een feature branch, PR of milestone
- Scant unit (ExUnit/whitebox), feature (Wallaby/greybox) en E2E (Playwright/blackbox) tests
- Geeft concreet advies aan developers op basis van de afgesproken 90/9/1 verdeling

## Ter review

Is de skill volledig en bruikbaar? Klopt de 90/9/1 verdeling en de beschrijving van whitebox/greybox/blackbox? Missen we nog conventies of stappen?

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: .claude/skills/eyra-test-scan/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+---
```
Comment (mellelieuwes) [inline .claude/skills/eyra-test-scan/SKILL.md:1]: Lijkt me goed om de skill in het engels te hebben. Zou je deze kunnen laten vertalen @irisvanderveen ?

**Review by mellelieuwes — CHANGES_REQUESTED** [review]

File: .claude/skills/eyra-test-scan/SKILL.md (hunk 1 above)
Comment (irisvanderveen) [reply .claude/skills/eyra-test-scan/SKILL.md:1]: @mellelieuwes ja dat is inderdaad een goed idee. Ik heb m aangepast naar Engels

File: .claude/skills/eyra-test-scan/SKILL.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+- Test lines added: <M>
+- Ratio: <M/N> (red flag if < 1:1 for core components)
+- New test distribution: <X>% unit / <Y>% feature / <Z>% E2E (target: 90/9/1)
+```
+
+## Important do's and don'ts
+
+- **Don't advise adding feature tests if the unit layer can do it better** — the test pyramid
+  says: focus on unit, feature is the middle ground, E2E is the happy path. Be strict.
+- **Flag a LiveComponent without a `live_isolated` test** as a recurring gap — this is often forgotten.
```
Comment (mellelieuwes) [inline .claude/skills/eyra-test-scan/SKILL.md]: Rename LiveComponent into LiveView

File: .claude/skills/eyra-test-scan/SKILL.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+- **Buggy-behavior tests that break after a fix** are normal — if you see these, no red flag, but do mention they'll need action after the fix.
+- **Mix tasks don't clash with `mix phx.server`** as long as they don't start an endpoint. `mix ecto.migrate`
```
Comment (irisvanderveen) [inline .claude/skills/eyra-test-scan/SKILL.md]: dit kan eruit

File: .claude/skills/eyra-test-scan/SKILL.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+  and `mix seed` are safe in parallel, `mix run priv/repo/seeds.exs` is not (it starts the whole app).
+- **No recommendation to run `mix test --cover` on the whole suite** — only on the touched
```
Comment (irisvanderveen) [inline .claude/skills/eyra-test-scan/SKILL.md]: deze kan er uit

File: .claude/skills/eyra-test-scan/SKILL.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+  paths. Otherwise too slow and not PR-relevant.
+- Path shifts: if `find core/test -name CLAUDE.md` returns less than before, check whether the content
```
Comment (irisvanderveen) [inline .claude/skills/eyra-test-scan/SKILL.md]: kan eruit

File: .claude/skills/eyra-test-scan/SKILL.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
+---
+name: eyra-test-scan
+description: |
+  Test gap analysis for a new feature, milestone, or branch in the Eyra Next platform (mono repo).
```
Comment (irisvanderveen) [inline .claude/skills/eyra-test-scan/SKILL.md]: Rewrite; Test gap analysis for a branch on the Eyra next platform

File: .claude/skills/eyra-test-scan/SKILL.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+  Scans changed production code and checks whether there are enough unit tests (ExUnit), feature
+  tests (Wallaby), and E2E tests (Playwright). Output is concrete advice for developers,
+  plus the question whether an E2E spec is still missing.
+triggers:
+  - test-scan
+  - /test-scan
```
Comment (irisvanderveen) [inline .claude/skills/eyra-test-scan/SKILL.md]: /ey-test-scan
andere triggers eruit

File: .claude/skills/eyra-test-scan/SKILL.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+  - test gap
+  - test gap analysis
+  - test coverage
+  - test coverage check
+---
+
+# Test Scan — gap analysis for Eyra Next
+
+This skill helps assess a new feature or milestone:
```
Comment (irisvanderveen) [inline .claude/skills/eyra-test-scan/SKILL.md]: new feature or milestone weg en ipv branch

File: .claude/skills/eyra-test-scan/SKILL.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+2. Whether an E2E spec is still needed (if so → `/e2e-create`).
+
+## Step 1 — Determine scope
+
+Ask (if not given): what is the scope?
+
+- **Branch** — a feature branch that is still open. Use `git diff develop..HEAD --stat`
+  to find changed files.
+- **PR number / URL** — `gh pr view <id>` to get the branch, then diff as above.
+- **Free-form** — a feature is mentioned by name. Then ask which modules that touches or search
```
Comment (irisvanderveen) [inline .claude/skills/eyra-test-scan/SKILL.md]: deze regel kan eruit

File: .claude/skills/eyra-test-scan/SKILL.md [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+## Step 2 — Read the conventions
+
+Before judging, **first read** the current test conventions in mono. Do a find — paths can shift:
+
+```bash
+find core/test -maxdepth 4 -name CLAUDE.md
+find core/test -maxdepth 4 -name AGENTS.md
+```
+
+Expected (at time of writing): `core/test/.claude/CLAUDE.md` (the 5 rules: atomic, guards first,
```
Comment (irisvanderveen) [inline .claude/skills/eyra-test-scan/SKILL.md]: Deze hele alinea kan weg

File: .claude/skills/eyra-test-scan/SKILL.md [hunk 11]
```diff
@@ -0,0 +1,153 @@
...
+let-it-crash, standard tuples, pattern match in function heads), `core/test/CLAUDE.md`
+(Wallaby selectors, signal isolation), `core/test/e2e/CLAUDE.md` (Playwright conventions). Read
+all of them before formulating advice.
+
+## Step 3 — List production code
```
Comment (irisvanderveen) [inline .claude/skills/eyra-test-scan/SKILL.md]: List code ipv list production code

File: .claude/skills/eyra-test-scan/SKILL.md [hunk 12]
```diff
@@ -0,0 +1,153 @@
...
+## Step 6 — Check E2E
+
+E2E tests the happy path as a user walks through it — one flow per critical system. Check `core/test/e2e/`:
+
+```bash
+ls core/test/e2e/*.spec.ts
+```
+
+If the PR introduces a new critical system (= a new top-level user journey) without an existing
+E2E spec → note this as a candidate for `/e2e-create`. If the system already has a spec →
```
Comment (irisvanderveen) [inline .claude/skills/eyra-test-scan/SKILL.md]: 'skill /e2e-create eruit halen

File: .claude/skills/eyra-test-scan/SKILL.md [hunk 13]
```diff
@@ -0,0 +1,153 @@
...
+add nothing, E2E is not an edge-case layer.
+
+## Step 7 — Write the report
+
+Output in this format:
+
+```
+## Test Scan — <scope>
```
Comment (irisvanderveen) [inline .claude/skills/eyra-test-scan/SKILL.md:108]: eerste drie van Summary weg 
En Unit test coverage toevoegen

**Review by mellelieuwes — APPROVED** [review]
